mirrord/layer/src/socket/ops.rs

```diff
@@ -258,8 +258,8 @@ pub(super) fn bind(
     }
 
     // To handle #1458, we don't ignore port `0` for UDP.
-    if (is_ignored_tcp_port(&requested_address, incoming_config)
```

Comment:
why?

Reply:
oh I realize now, but that's not a solution -this warning can also happen with tcp.. (just more rare)

Reply:
I think it should still fail if it was ignored, since we try to match local to remote ports when it comes to binding..

Reply:
> I think it should still fail if it was ignored, since we try to match local to remote ports when it comes to binding..

I don't understand @aviramha

Reply:
Perhaps clarify your original intent since I think we might not be talking about the same thing

Reply:
Nevermind. Is [this](https://github.com/metalbear-co/mirrord/pull/2813/commits/a8e3dee2d12d3b7d2af466f821e54190d0e8a2fa) a solution? - moved the warning to `listen` so that we don't show it if the user app binds a port that is ignored for incoming but then uses it for outgoing.

Reply:
I'm okay with this, would get another review tho.